### PR TITLE
:hammer: Add some notifications functions to the API

### PR DESCRIPTION
**Pull request by Zeleeer**
Afin de pouvoir contribuer à mon niveau, j'espère que ces lignes seront utiles :)

Bon courage à tous.

### CODE REVIEW

**Comment by PABLO-1610** [comment]
Hi, thanks for contributing.

First of all, all of your PRs need to be written in English.
And in the API, all functions are separated in files.

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Nice code, but you don't respect the code format established in the project.

**Comment by Zeleeer** [comment]
Hi, 

Thanks for the returns :)

I try to make them with your code format

File: src/api/players/client/helpNotif.lua [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---helpNotif
+---@param text string
+---@public
+
+function _NCS:helpNotif(text)
+    AddTextEntry('HELP', text)
+    DisplayHelpTextThisFrame('HELP', false)
+end
```
Comment (PABLO-1610) [inline src/api/players/client/helpNotif.lua:8]:
```suggestion
---showHelpNotification
---@param text string
---@return void
---@public
function _NCS:showHelpNotification(text)
    AddTextEntry("HELP", text)
    DisplayHelpTextThisFrame("HELP", false)
end
```

File: src/api/players/client/iconNotif.lua [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+---@param colorBg number
+---@public
+
+function _NCS:iconNotif(icon, type, sender, title, text, colorBg)--( logo, 1, expediteur, objet, message)
+    if colorBg then ThefeedNextPostBackgroundColor(colorBg) end
+	SetNotificationTextEntry("STRING");
+	AddTextComponentString(text);
+	SetNotificationMessage(icon, icon, true, type, sender, title, text);
+	DrawNotification(false, true);
+end
```
Comment (PABLO-1610) [inline src/api/players/client/iconNotif.lua:16]:
```suggestion
---showAdvancedNotification
---@param icon string
---@param type number
---@param sender string
---@param title string
---@param text string
---@param colorBg number
---@return void
---@public
function _NCS:showAdvancedNotification(icon, type, sender, title, text, colorBg)
    if (colorBg) then 
        ThefeedNextPostBackgroundColor(colorBg) 
    end
    SetNotificationTextEntry("STRING")
    AddTextComponentString(text)
    SetNotificationMessage(icon, icon, true, type, sender, title, text)
    DrawNotification(false, true)
end
```

File: src/api/players/client/missionNotif.lua [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+---@param text string
+---@param time number
+---@public
+
+function _NCS:missionNotif(text, time)
+	ClearPrints()
+	BeginTextCommandPrint('STRING')
+	AddTextComponentSubstringPlayerName(text)
+	EndTextCommandPrint(time, true)
+end
```
Comment (PABLO-1610) [inline src/api/players/client/missionNotif.lua:11]:
```suggestion
---showMissionNotification
---@param text string
---@param time number
---@return void
---@public
function _NCS:showMissionNotification(text, time)
	ClearPrints()
	BeginTextCommandPrint("STRING")
	AddTextComponentSubstringPlayerName(text)
	EndTextCommandPrint(time, true)
end
```

File: src/api/players/client/simpleNotif.lua [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+---@param text string
+---@param colorBg number
+---@public
+
+function _NCS:simpleNotif(text, colorBg)
+    if colorBg then ThefeedNextPostBackgroundColor(colorBg) end
+	AddTextEntry('SowNotifSimple', text)
+	BeginTextCommandThefeedPost('SowNotifSimple')
+	EndTextCommandThefeedPostTicker(false, false)
+end
```
Comment (PABLO-1610) [inline src/api/players/client/simpleNotif.lua:11]:
```suggestion
---showNotification
---@param text string
---@param colorBg number
---@return void
---@public
function _NCS:showNotification(text, colorBg)
    if (colorBg) then 
        ThefeedNextPostBackgroundColor(colorBg) 
    end
    AddTextEntry('SowNotifSimple', text)
    BeginTextCommandThefeedPost('SowNotifSimple')
    EndTextCommandThefeedPostTicker(false, false)
end
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution, but please consider moving thoses files into `src/api/notifications`

**Pull request closed without merging**